private/model/api/operation.go

```diff
@@ -127,8 +127,9 @@ func (c *{{ .API.StructName }}) {{ .ExportedName }}Request(` +
 {{ range $_, $err := .ErrorRefs -}}
 	{{ $errDoc := $err.IndentedDocstring -}}
 //   * {{ $err.Shape.ErrorCodeName }} "{{ $err.Shape.ErrorName}}"
-{{ if $errDoc -}}
-{{ $errDoc }}{{ end }}
+{{ if $err.Docstring -}}
+{{ $errDoc }}
```

Comment:
Should the var `$errDoc` be moved here instead of being a variable above?

```go
{{ if $err.Docstring -}}
{{ $errDoc := $err.IndentedDocstring }}
{{ end -}}
```